policyengine_us/parameters/gov/states/al/tax/income/deductions/itemized/excluded_deductions.yaml

```diff
@@ -0,0 +1,15 @@
+description: Itemized deductions excluded from Alabama.
+values:
+  2021-01-01:
+    - casualty_loss_deduction
+    - long_term_care_premium_deduction
+
+
+metadata:
+  unit: list
+  label: Itemized deductions excluded from Alabama.
+  reference:
+    - title: 2022 Schedule A (Form 1040)
```

Comment:
please also use this source for all parameters (point to exact section of the law document where you can find the parameter) f.e. - here: (13) 
https://law.justia.com/codes/alabama/2021/title-40/chapter-18/article-1/section-40-18-15/